src/Configure.ts

```diff
@@ -21,12 +21,14 @@ export interface IConfiguration {
   adapter?: IAdapterFactory<any>;
   serializer?: ISerializerFactory<any, any>;
   mapper?: IMapperFactory<any>;
+  preferPatchOverPut?: boolean;
 }
 
 const defaultConfiguration: Partial<IConfiguration> = {
   adapter: RestAdapter,
   serializer: RestSerializer,
   mapper: Mapper,
+  preferPatchOverPut: false,
```

Comment:
Wont matter for us because we will set it to preferPatchOverPut = true in SIS Web but I figure not all consumers of redux-data-service will have baked in support for patches and will want things to work out of the box.

Reply:
Agreed 👍